test/tpmclient/tpmclient.int.cpp

```diff
@@ -6479,6 +6480,11 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
     TestDictionaryAttackLockReset();
 
     TestCreate();
+    rval = Tss2_Sys_FlushContext( sysContext, loadedSha1KeyHandle );
```

Comment:
We should start a larger conversation w/r to coding style. The original author used an undocumented style that's very foreign to OSS code. We need to document the style that we want to use going forward and set some rules around how we handle the migration. Personally when I add *new* code I use the style documented in the tools and tabrmd repo. When fixing up *existing* code I've tried to mimic the style used previously as best I can.